README.md

```diff
@@ -1,29 +1,48 @@
 # cdist
 
-**cdist** is a usable configuration management system.
+This is [community maintained](https://github.com/cdist-community)
+fork of Ungleich's [cdist](https://github.com/ungleich/cdist)
+(after [f061fb168ddacc894cb6e9882ff5c8ba002fadd8](https://github.com/ungleich/cdist/commit/f061fb168ddacc894cb6e9882ff5c8ba002fadd8)).
 
-It adheres to the [**KISS principle**](https://en.wikipedia.org/wiki/KISS_principle)
-and is being used in small up to enterprise grade environments.
+Work is split between three repositories:
 
-For more information have a look at [**homepage**](https://cdi.st)
-or at **``docs/src``** for manual in **reStructuredText** format.
+* [cdist](https://github.com/cdist-community/cdist) - main implementation of **cdist core**.
+* [cdist-conf](https://github.com/cdist-community/cdist-conf) - **essential** explorers and types.
+* [cdist-extra](https://github.com/cdist-community/cdist-extra) - **non-essential** explorers, types, scripts, tools etc.
 
-## Contributing
+What's the difference between essential and non-essential? Explorers and types,
```

Comment:
We should sync this up with cdist-community/cdist-conf#2 once we agreed on a definition.

Reply:
Yes, initial section of README (until getting started) should be used for other 2 repos as well.

Reply:
Created PRs for other two repos.